spec/facter/facts/freebsd/memory/system/capacity_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+describe Facts::Freebsd::Memory::System::Capacity do
+  describe '#call_the_resolver' do
+    it 'returns a fact' do
+      expected_fact = double(Facter::ResolvedFact, name: 'memory.system.capacity', value: '15.53%')
+
+      allow(Facter::Resolvers::Freebsd::SystemMemory).to receive(:resolve).with(:capacity).and_return('15.53%')
+      allow(Facter::ResolvedFact).to receive(:new).with('memory.system.capacity', '15.53%').and_return(expected_fact)
+
+      fact = Facts::Freebsd::Memory::System::Capacity.new
+      expect(fact.call_the_resolver).to eq(expected_fact)
+    end
+  end
+end
```

Comment:
This test should be written using the new template for writing unit test for facts.